src/main/js/main_html5.js

```diff
@@ -791,18 +805,24 @@ require("../../../html5-common/js/utils/environment.js");
         //If the captions are in-stream, we just need to enable them; Otherwise we must add them to the video ourselves.
         if (captions.inStream == true && _video.textTracks) {
           for (var i = 0; i < _video.textTracks.length; i++) {
-            if (((OO.isSafari || OO.isEdge) && isLive) || _video.textTracks[i].kind === "captions") {
+            if (
+              (((OO.isSafari || OO.isEdge) && isLive) || _video.textTracks[i].kind === "captions") &&
```

Comment:
I decided to keep this part of the old code in order to minimize the impact of this change, but my guess is that whoever wrote this was trying to localize this feature to the platforms where it was known that there could be in-stream captions in order to avoid handling unexpected cases. Eventually we should switch to just detecting these tracks regardless or where they show up, but I didn't want to introduce any new behavior with this change.